src/util/AuthTokenListener.js

```diff
@@ -155,34 +157,33 @@ async function authTokenListener(client, guildId, steamId, firstTime = false) {
                         pairingServer(client, guildId, data, hoster);
                     } break;
 
-                    //case 'entity': {
-                    //    switch (data.entityName) {
-                    //        case 'Smart Switch': {
-                    //            client.log('AuthToken',
-                    //                `GuildID: ${guildId}, SteamID: ${steamId}, pairing: entity: Switch`);
-                    //            pairingEntitySwitch(client, guild, full, data, body);
-                    //        } break;
-
-                    //        case 'Smart Alarm': {
-                    //            client.log('FCM Host',
-                    //                `GuildID: ${guild.id}, SteamID: ${hoster}, pairing: entity: Smart Alarm`);
-                    //            pairingEntitySmartAlarm(client, guild, full, data, body);
-                    //        } break;
-
-                    //        case 'Storage Monitor': {
-                    //            client.log('FCM Host',
-                    //                `GuildID: ${guild.id}, SteamID: ${hoster}, pairing: entity: Storage Monitor`);
-                    //            pairingEntityStorageMonitor(client, guild, full, data, body);
-                    //        } break;
-
-                    //        default: {
-                    //            client.log('FCM Host',
-                    //                `GuildID: ${guild.id}, SteamID: ${hoster}, ` +
-                    //                `pairing: entity: other\n${JSON.stringify(full)}`);
-                    //        } break;
-                    //    }
-
-                    //} break;
+                    case 'entity': {
+                        switch (data.entityName) {
+                            case 'Smart Switch': {
+                                client.log('AuthToken',
+                                    `GuildID: ${guildId}, SteamID: ${steamId}, pairing: entity: Switch`);
+                                pairingEntitySwitch(client, guildId, data);
+                            } break;
+
+                            case 'Smart Alarm': {
+                                client.log('FCM Host',
```

Comment:
Change these from FCM Host -> AuthToken.
Also change from hoster -> steamId in same message.